README.md

```diff
@@ -498,6 +500,13 @@ which contains the `luarocks` stderr output, among other logs.
 - [luarocks.nvim](https://github.com/vhyrro/luarocks.nvim):
   Adds basic support for installing lua rocks to [lazy.nvim](https://github.com/folke/lazy.nvim)
 
+
+## Other neovim plugin managers
+
+- [lazy.nvim](https://github.com/folke/lazy.nvim): started luarocks support in
+  version [11.X](https://lazy.folke.io/news#11x)
+- [vim-plug](https://github.com/junegunn/vim-plug)
+
```

Comment:
note: It feels a bit strange to me mentioning only lazy.nvim and vim-plug here.
We should either move the lazy.nvim mention to the above section.
Or we should mention additional plugin mangers.
Some that come to mind:

- [pckr](https://github.com/lewis6991/pckr.nvim)
- [paq-nvim](https://github.com/savq/paq-nvim)
- [mini.deps](https://github.com/echasnovski/mini.deps)

Reply:
absolutely. Do you think I should merge this section with the previous ?

Reply:
I think I like the separation though because it makes it clear those are "competition"

Reply:
similar type of rocks.nvim. Could be a subsection of the previous one as well.

Reply:
I think two sections are fine.